### DISCUSSION TITLE
Tenant name 30 character limitation

### DISCUSSION THREAD

**Discussion by nomaxpi** [netbox-community/netbox#5589] [Q&A] [answered]
I am trying to use netbox via the API so I can keep it and our CRM in sync when new customers are added. Obviously I have to shorten anything over 30 characters and it's a little hard to predict what will appear in our CRM when someone new is setup. Is there any way to allow for more characters so I don't have to write a crazy amount of word shortening code? Some names come out ok but I can't predict everything someone is going to use as a customer name so some come out unrecognizable which makes it useless to someone using the GUI once the record has been created via the API.

TIA
Max

**Comment by jeremystretch** [comment, marked as answer]
This limit was increased to 100 characters in the v2.10.0 release. If you prefer to stick with an earlier release for now, you can use an internal code for the tenant name and put it's full name in the description.

**Reply by nomaxpi** [reply]
Ah, that's great news. I thought I read through the versioning release notes but didn't see that specific enhancement. We'll certainly upgrade to the new release. Thanks!
